### PR TITLE
Hide empty header

### PR DESCRIPTION
**Pull request by vasileios-karavasilis-sociomantic**
The functionality has been implemented (I am not sure if this is the best approach, any comments are welcome).
I will add some test next week.

Fixes: #120.

### CODE REVIEW

File: src/core/build.js [hunk 1]
```diff
@@ -281,6 +283,12 @@ const build = {
 
             if ( dataObj.header )
             {
+                // Ignore empty sections.
+                if ( dataObj.data === undefined || dataObj.data.length == 0 )
```
Comment (vasileios-karavasilis-sociomantic) [inline src/core/build.js]: The record exist in the `dataObj`, but no in the `sections`. We must keep this in `sections` also.

**Comment by vasileios-karavasilis-sociomantic** [comment]
Updated. I have added some unit tests. I have also tested it manually and it seems to work.

**Comment by mousemke** [comment]
was there a reason this didnt get merged or was it just forgotten?

**Comment by vasileios-karavasilis-sociomantic** [comment]
Not from my side. But I can not do this as I don't have write permissions over this repository.

**Pull request merged**